docs/developer_guide/web_app_guide.md

```diff
@@ -0,0 +1,188 @@
+
+## Overview
+This guide describes a quick and convenient method for configuring and hosting a monero-javascript web application server using the monero-javascript web application starter script. The guide also demonstrates a practical use case by converting the offline wallet generator from [part 1 the getting started guide](https://moneroecosystem.org/monero-javascript/docs/developer-guide/getting_started.md)) into a web application.
+
+## Generating a monero-javascript web application project
+### Required software
+The web application starter script relies on two external programs:
+* [Python (version 3)](https://github.com/timetherewere/monero-javascript/blob/master/docs/developer_guide/prerequisite_installation_guides#python)
+* [Node.js and the node package manager (npm)](https://github.com/timetherewere/monero-javascript/blob/master/docs/developer_guide/prerequisite_installation_guides#node.js_and_npm)
+  _If you followed the [getting started guide](https://moneroecosystem.org/monero-javascript/docs/developer-guide/getting_started.md), you should already have intalled node.js._
+
+Click the links above for more information on installing and configuring Python, node.js, and npm.
+
+### Configuring the project directory
+Monero-javascript provides a script that automatically downloads the files and configures the directories you need to host a functioning monero-javascript web application server. To automatically download and run the script:
+
+1. Create a new folder for the project (e.g. `mkdir ~/monero-javascript_web_app`)
+2. Enter the new directory (`cd ~/monero-javascript_web_app`)
+3. Download and run the script:
+
+`$ bash <(curl -sL https://raw.githubusercontent.com/woodser/xmr-sample-app/master/web_template_script.sh)`
+
+<!--
+#### Windows
+1. Create a new folder for the project (e.g. `mkdir C:\Users\[username]\monero_javascript_web_app`)
+2. Enter the new directory (`cd C:\Users\[username]\monero_javascript_web_app`)
+3. Download and run the script:
+```
+powershell -Command "(new-object net.webclient).DownloadString('https://raw.githubusercontent.com/woodser/xmr-sample-app/add_starter_scripts/webTemplateScript.bat')"
+webTemplateScript.bat
+```
+-->
+Alternately, you can manually download and run the shell scripts.
+[Linux/Mac](https://github.com/timetherewere/monero-javascript/raw/include_compressed_template/web_template_script)
+<!--
+[Windows](https://github.com/timetherewere/monero-javascript/raw/include_compressed_template/web_template_script.bat)
+-->
+The script will configure the project folder and serve the starter web application on port 9100. Open a web browser and type "localhost:9100" in the URL bar to view the application.
+
+_Note_:
+* Unlike create-react-app, the xmr-sample-app script will not automatically open a web browser. You must open the browser and navigate to "localhost:9100" manually.
```

Comment:
First reference to create-react-app.  Remove or add hyperlink and explanation that web_template_script is similar to create-react-app?